src/components/forums/topic/SharePost.tsx

```diff
@@ -0,0 +1,49 @@
+import { useState } from 'react';
+
+import { Chain } from '../../../assets';
+import { MessageType } from './../../common';
+import { Notification } from '../../../components/forums';
+import { NOTIFICATION_BANNER_TIMEOUT } from '../../../utils/consts';
+
+interface SharePostProps {
+  postAddress: string;
+}
+
+const SharePost = ({ postAddress }: SharePostProps): JSX.Element => {
```

Comment:
Maybe define it as `export function SharePost` just so we define all the components the same way

Reply:
Okey, I changed all components that I defined for this form